Comments/ClientApp/src/components/GenerateCode/GenerateCode.scss

```diff
@@ -0,0 +1,43 @@
+.organisation-codes {
+  &__row {
+    display: flex;
+    align-items: center;
+  }
+
+  &__name, &__generate {
+    flex: 0 0 50%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    p, button {
+      margin: 0;
+    }
+  }
+}
+
+.tag {
+  &.tag--copied {
+    background-color: #e6faff;
+    color: #000;
+    padding: 0 15px;
```

Comment:
re: the 15px spacing..
does that look good on all device sizes? maybe it should use a sass variable from here:
https://nice-digital.github.io/nice-design-system/foundations/spacing/#introduction